platform/viewer/src/App_bk.js

```diff
@@ -1,319 +0,0 @@
-import React, { Component } from 'react';
```

Comment:
What is this file? hehe backup?

Reply:
cleanup?

Reply:
This file is a copy of the current (v1) `App.js` from the Viewer application.
Removing because it was left in history but it's not needed to be here.